inspirehep/modules/search/walkers/elasticsearch.py

```diff
@@ -90,6 +90,11 @@ def visit(self, node):
     @visitor(Value)
     def visit(self, node):
         def query(keyword):
+            # FIXME: This is a temporary hack that should be removed when
+            # nested keywords search is implemented.
+            if str(node.value).startswith('recid:'):
+                node.value = node.value[len('recid:'):]
+
```

Comment:
We will have to drop this specific behavior once we enable second order operators...